test/jdk/ProblemList.txt

```diff
@@ -834,6 +834,7 @@ java/awt/Checkbox/CheckboxNullLabelTest.java 8340870 windows-all
 java/awt/dnd/WinMoveFileToShellTest.java 8341665 windows-all
 java/awt/Menu/MenuVisibilityTest.java 8161110 macosx-all
 java/awt/Modal/NativeDialogToFrontBackTest.java 7188049 windows-all,linux-all
+java/awt/Cursor/CursorDragTest/ListDragCursor.java 8359061 macosx-all
```

Comment:
Do you mean to use [7177297](https://bugs.openjdk.org/browse/JDK-7177297) in problemlist entry?

Reply:
Doesn't seem right either. There's a warning about this from the Skara bot.

Reply:
Makes sense. Thanks. I have updated the bugID in the PL.